service/framework.go

```diff
@@ -463,6 +472,8 @@ func NewVersionedResources(config Config, k8sClient kubernetes.Interface, awsCon
 		"1.0.0": legacyResources,
 		"2.0.0": resourcesV2,
 		"2.0.1": resourcesV2_0_1,
+		// 2.0.2 fixes missing region in host account credentials, the change only affects service/framework.go
```

Comment:
This makes sense to me since the fix is only in framework.go.

Reply:
I'd say we shouldn't even introduce a new version bundle if this affects only operator's code.

Reply:
If it is a bug that affects the customer we have to because this is the whole point of reliable state management in our infrastructure and the communication of state and technology towards the customer. We must never sneak in something secretly that affects the customer's observable behaviour.